recipes/WHAMandWHAMR/enhancement/train.py

```diff
@@ -80,16 +80,17 @@ def compute_forward(self, mix, targets, stage, noise=None):
                     else:
                         mix = targets.sum(-1)
 
-                    noise = noise.to(self.device)
-                    len_noise = noise.shape[1]
-                    len_mix = mix.shape[1]
-                    min_len = min(len_noise, len_mix)
+                    if noise is not None:
```

Comment:
In what context can it be None? Is it a params?

Reply:
@ycemsubakan please take a look - same here, I think some tests failed if this was not in